tests/assets/recipes/baked/daylight-factor.yaml

```diff
@@ -57,247 +62,257 @@ flow:
       platform:
       - grasshopper
       handler:
-      - annotations: {}
-        type: IOAliasHandler
+      - type: IOAliasHandler
+        annotations: {}
         language: python
         module: honeybee_rhino.handlers
         function: hb_model_to_hbjson
-      - annotations: {}
-        type: IOAliasHandler
+      - type: IOAliasHandler
+        annotations: {}
         language: csharp
         module: HBRhino.Model
         function: HBModelToJSON
-    - annotations: {}
-      type: DAGGenericInputAlias
+    - type: DAGGenericInputAlias
+      annotations: {}
       name: model
       description: null
       default: null
       spec: null
       platform:
       - rhino
       handler:
-      - annotations: {}
-        type: IOAliasHandler
+      - type: IOAliasHandler
+        annotations: {}
         language: dotnet
         module: HBRhino.Model
         function: ExportRhinoModel
     extensions: null
-  - annotations: {}
-    type: DAGStringInput
+  - type: DAGStringInput
+    annotations: {}
     name: radiance-parameters
     description: The radiance parameters for ray tracing
     default: -I -ab 2 -h
     spec: null
     alias: []
-  - annotations: {}
-    type: DAGIntegerInput
+  - type: DAGIntegerInput
+    annotations: {}
     name: sensor-grid-count
     description: The maximum number of grid points per parallel execution
     default: null
     spec: null
     alias: []
   outputs:
-  - annotations: {}
-    type: DAGIntegerOutput
+  - type: DAGIntegerOutput
+    annotations: {}
     name: average
     description: null
     from:
-      annotations: {}
       type: TaskReference
+      annotations: {}
       name: post-process
       variable: daylight-factor-average
     alias: []
-  - annotations: {}
-    type: DAGFolderOutput
+  - type: DAGFolderOutput
+    annotations: {}
     name: data
     description: null
     from:
-      annotations: {}
       type: TaskReference
+      annotations: {}
       name: post-process
       variable: post-process-folder
     alias: []
-  name: 8b959cb7cb3b5f24abd8ac64c03b44c457b318cc9494a7eeca514638edd43e34/main
+  name: 91d2c0727a0d749c710d71f5cbcee77cbc6a0cc2af29596f6ea236497eccf19b/main
```

Comment:
In Pollination's returned object, it is just "main" for the this entry flow, can we remove this digest for this main?

Reply:
it is main in the regular recipe.

Reply:
You should check the one under the valid folder. That doesn't have this.